### PR TITLE
UITOOL-397/implement simple feature flags

### PR DESCRIPTION
**Pull request by aiheon**
### Proposed Changes

https://coveord.atlassian.net/browse/UITOOL-397

Using this ---> https://github.com/jamesmfriedman/flagg

This is a simple implementation for feature flags in Plasma, using the sessionStorage for the browser.

The need for Feature flags in Plasma is coming from the Search Bar development. The process is long and we want the possibility of splitting the work in multiple prs without showing the bar during it's development.

### Potential Breaking Changes

None

### Acceptance Criteria

-   [ ] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/plasma/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

**Review by mikedidomizio — APPROVED** [review]
Cool, tested and can verify it works

File: packages/website/package.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
         "codemirror": "5.63.3",
         "d3": "3.5.17",
         "faker": "4.1.0",
+        "flagg": "1.1.2",
```
Comment (gdostie) [inline packages/website/package.json:34]: FLA-double G
image

File: packages/website/src/OneDemoToRuleThemAll.tsx [hunk 2]
```diff
@@ -24,9 +25,17 @@ const Header = () => (
...
+                To toggle the feature flag, copy and paste those commands in the dev tool console:
+                sessionStorage.setItem('ff_plasma-search-bar', true) to show the bar
+                sessionStorage.setItem('ff_plasma-search-bar', false) to hide the bar
+                You need to reload the page for it to take effect.
+             */}
+            {FeatureFlags.get('plasma-search-bar') && (
+                <EngineProvider>
+                    <StandaloneSearchBar id="header" />
+                </EngineProvider>
+            )}
```
Comment (gdostie) [inline packages/website/src/OneDemoToRuleThemAll.tsx:38]: Can we move this inside a component so that it remains transparent to this file?

**Review by gdostie — APPROVED** [review]

File: packages/website/package.json (hunk 1 above)
Comment (aiheon) [reply packages/website/package.json:34]: :laughing:

File: packages/website/src/OneDemoToRuleThemAll.tsx (hunk 2 above)
Comment (aiheon) [reply packages/website/src/OneDemoToRuleThemAll.tsx:38]: I'll try :eyes:

**Pull request merged**